src/t_env/env_del.c

```diff
@@ -6,52 +6,61 @@
 /*   By: minakim <minakim@student.1337.ma>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/08/08 13:22:59 by minakim           #+#    #+#             */
-/*   Updated: 2023/08/14 17:15:20 by minakim          ###   ########.fr       */
+/*   Updated: 2023/09/02 17:38:05 by minakim          ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 #include "../../include/minishell.h"
 #include "../../libft/include/libft.h"
 
-int	env_reset_node(t_env *node_to_delete)
+void	env_del(t_env *target)
```

Comment:
The functions in this file have been slightly modified to be consistent with the functions you wrote and to use void to remove obsolete return values.